packages/ckeditor5-core/src/editor/editorconfig.jsdoc

```diff
@@ -297,3 +298,28 @@
  *
  * @member {String} module:core/editor/editorconfig~EditorConfig#placeholder
  */
+
+/**
+ * The editor UI configuration.
```

Comment:
I miss a link to toolbar configuration guide where this option will be explained in a context of what it affects.

Reply:
The idea here is to not connect it with the toolbar at all. It may affect anything, not just the toolbar.

Also, the `ui` namespace may be used to add any other `ui` related config stuff in the future that may not have anything to do with the toolbar at all.

Since we are completely removing the `vieportTopOffset` from the toolbar config docs, it may be confusing IMHO.

Reply:
I linked `ui.viewportOffset` config (in a note) to both places that use the `top` value and vice-versa. I liked both of these places to the `ui` config.

If you are about to build the docs locally, you'll find these under links:

Config: http://127.0.0.1:8080/ckeditor5/29.1.0/api/module_core_editor_editorconfig-EditorConfig.html#member-ui

Inline Editor: http://127.0.0.1:8080/ckeditor5/29.1.0/api/module_editor-inline_inlineeditoruiview-InlineEditorUIView.html#member-viewportTopOffset

Sticky Panel View: http://127.0.0.1:8080/ckeditor5/29.1.0/api/module_ui_panel_sticky_stickypanelview-StickyPanelView.html#member-viewportTopOffset